README.md

```diff
@@ -89,6 +90,22 @@ cd my-blog
 gem install
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/228943689-38a9733b-72b8-4b45-9067-948db5b48756.png)

**[OPTIONAL]**

- [ ] It would be nice if you add some good features in your Future Features section to make excitement for the viewers of your application :+1: and repository :blush: You can add such as

* Optimized Database
* Better UI Experience